frontend/resourceadm/pages/AboutResourcePage/AboutResourcePage.tsx

```diff
@@ -275,24 +274,20 @@ export const AboutResourcePage = ({
           onChange={(isChecked: boolean) => handleSave({ ...resourceData, visible: isChecked })}
           toggleTextTranslationKey='resourceadm.about_resource_visible_show_text'
         />
-        {shouldDisplayFeature('resourceAccessLists') && (
-          <>
-            <ResourceSwitchInput
-              label={t('resourceadm.about_resource_limited_by_rrr_label')}
-              description={t('resourceadm.about_resource_limited_by_rrr_description')}
-              value={resourceData.limitedByRRR ?? false}
-              onFocus={() => setTranslationType('none')}
-              onChange={(isChecked: boolean) =>
-                handleSave({ ...resourceData, limitedByRRR: isChecked })
-              }
-              toggleTextTranslationKey='resourceadm.about_resource_use_rrr_show_text'
-            />
-            {resourceData.limitedByRRR && (
-              <div data-testid='rrr-buttons'>
-                <AccessListEnvLinks />
-              </div>
-            )}
-          </>
+        <ResourceSwitchInput
+          label={t('resourceadm.about_resource_limited_by_rrr_label')}
+          description={t('resourceadm.about_resource_limited_by_rrr_description')}
+          value={resourceData.limitedByRRR ?? false}
+          onFocus={() => setTranslationType('none')}
```

Comment:
These are a bit tricky to test, since they are callbacks from the child component and would require a test to know how to trigger the callbacks from interaction with the child component. The untested lines in AboutResourcePage.tsx should rather be tested after a (future) refactor to make the file more testable 😄